livedata/src/main/java/io/getstream/chat/android/livedata/controller/QueryChannelsControllerImpl.kt

```diff
@@ -24,6 +24,10 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.SupervisorJob
 
+private const val MESSAGE_LIMIT = 10
```

Comment:
Nit: It's a matter of taste, but such constant looks better in the companion object of such class

Reply:
Same, having them nested in the companion object (or just another named object) inside the class is more natural for me, makes the constants belong specifically to the class.

(And it makes the icon for the file looks nicer in the Project pane of the IDE)

Reply:
When this constant is used outside of the class, I see the point to use the companion object and have the code more accessible/ordered, but for private constants that are used only inside of this file, I don't see any benefit, it looks more like "Java-Style"

Reply:
In theory, we could have multiple classes in the same file. If only one of those uses a given constant, I'd have place the constants inside the class, to have it exist in the smallest scope possible.

Placing it inside the class, even while there's only one in the file, puts it in a narrower scope than having it at the file level, even if private.